### PR TITLE
Basic scripts for Halloween domains

### PR DESCRIPTION
**Pull request by misslivirose**
Test plan: 

- [x] go to haunted forest domain at /6.2, -8.6, -76.3. You should hear 'help me' if you walk around in the space 
- [x] you should see a spider appear in that area after some haunted music plays
- [x] in the haunted forest domain, head to /-3.0, -12.8, -19.4
- [x] you should see a scary face appear and hear a laugh

### CODE REVIEW

File: Prototyping/Halloween/Spider/HelpMeZoneServer.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }, 
+        unload: function() {
+
+        }, 
+        startZoneEffect : function() { 
+            if (audioElement.downloaded) {
+                Audio.playSound(audioElement, {
+                    position: AUDIO_POSITION,
+                    volume: Math.random(),
+                    pitch: 1
```
Comment (MiladNazeri) [inline Prototyping/Halloween/Spider/HelpMeZoneServer.js]: Not used, unless something was meant to change pitch

File: Prototyping/Halloween/Spider/ScaryZoneServer.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        }, 
+        unload: function() {
+
+        }, 
+        startZoneEffect : function() { 
+            if (audioElement.downloaded) {
+                Audio.playSound(audioElement, {
+                    position: AUDIO_POSITION,
+                    volume: Math.random(),
+                    pitch: 1,
```
Comment (MiladNazeri) [inline Prototyping/Halloween/Spider/ScaryZoneServer.js]: same here, not a big deal, just one of the only things i've noticed.

File: Prototyping/Halloween/Spider/HelpMeZoneServer.js (hunk 1 above)
Comment (misslivirose) [reply Prototyping/Halloween/Spider/HelpMeZoneServer.js]: ty!

**Comment by misslivirose** [comment]
@AndySeattle - These scripts are in the forest right now. Do they need a specific additional test pass or can we approve this for what's in there at the moment?

**Comment by AndySeattle** [comment]
@misslivirose I cannot access haunter forest still, are we going to move the content to a different domain? Did you get a copy of the content?

**Comment by AndySeattle** [comment]
Hey @misslivirose I am not hearing "help me" on the new forest domain, also i do not see the spider spawn in its spot till im well behind it. Did the zones get moved around perhaps?

**Comment by misslivirose** [comment]
@AndySeattle - The server script wasn't running for some reason - I restarted it and it's appearing now

**Pull request merged**